application/src/main/java/org/togetherjava/tjbot/features/help/HelpThreadCreatedListener.java

```diff
@@ -90,9 +91,17 @@ private void handleHelpThreadCreated(ThreadChannel threadChannel) {
         SERVICE.schedule(createMessages, 5, TimeUnit.SECONDS);
     }
 
+    private RestAction<Message> createAIResponse(ThreadChannel threadChannel) {
+        RestAction<Message> originalQuestion =
+                threadChannel.retrieveMessageById(threadChannel.getIdLong());
+        return originalQuestion.flatMap(
+                message -> helper.constructChatGptAttempt(threadChannel, message.getContentRaw()));
+    }
+
     private RestAction<Message> createMessages(ThreadChannel threadChannel) {
         return sendHelperHeadsUp(threadChannel).flatMap(Message::pin)
-            .flatMap(any -> helper.sendExplanationMessage(threadChannel));
+            .flatMap(any -> helper.sendExplanationMessage(threadChannel))
+            .flatMap(any -> threadChannel.retrieveMessageById(threadChannel.getIdLong()));
```

Comment:
y retrieve the message here and not do anything with it?

Reply:
Looks like old code. Thanks for pointing it out. Removed.